dao/src/main/java/com/garret/movies/dao/entity/Genre.java

```diff
@@ -0,0 +1,16 @@
+package com.garret.movies.dao.entity;
+
+import lombok.Data;
+
+import javax.persistence.*;
+
+@Data
+@Entity
+@Table(name = "genre")
+public class Genre {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String value;
```

Comment:
same here, I suppose male/female, if so this should be enum

Reply:
@mmarusich no, here is name of genre (action, comedy etc)

Reply:
Ok but those values are static, working with enum will be much easier

Reply:
@mmarusich I agree, but I don’t know the names of all genres

Reply:
OK, this is an issue of this API specifically that it missing documentation, but we should not stick to it and do design properly. Add whatever you know, others will add later once you receive it and fail, this is not best approach, but again this is only because we are missing documentation